client/my-sites/customer-home/index.js

```diff
@@ -0,0 +1,18 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import { navigation, siteSelection, sites } from 'my-sites/controller';
+import home from './controller';
+import { makeLayout, render as clientRender } from 'controller';
+
+export default function() {
+	page( '/home', siteSelection, sites, makeLayout, clientRender );
+
+	page( '/home/:site_id', siteSelection, navigation, home, makeLayout, clientRender );
```

Comment:
Does this need snake case for any reason here and not `siteId`?

Not a biggie, just Calypso convention

Reply:
Not really sure where it came from but I've changed it now. Thanks!